hops/featurestore_impl/core.py

```diff
@@ -1255,6 +1279,12 @@ def _do_get_training_dataset_path(training_dataset_name, featurestore_metadata,
                                                             training_dataset_version)
     hdfs_path = training_dataset.location + \
                 constants.DELIMITERS.SLASH_DELIMITER + training_dataset.name
+
+    # If the path starts with `hopsfs` remove it so we can pass it 
+    # to TensorFlow
+    if hdfs_path.startswith("hopsfs://"):
+        hdfs_path = hdfs_path[hdfs_path.index("/", 9):]
```

Comment:
I think this should be `8` since it starts counting from zero.

Reply:
No, this endpoint returns also the IP and port of the namenode. If you set 8 then you would get something like "/10.0.2.15:8020/path" which is not a valid path.

Reply:
This code should be already on master, need to rebase the PR to make it go away :)